packages/dbml-core/src/parse/dbml/src/lib/analyzer/symbol/symbols.ts

```diff
@@ -0,0 +1,106 @@
+import SymbolTable from './symbolTable';
+import { SyntaxNode } from '../../parser/nodes';
+
+export enum SymbolKind {
+  SCHEMA = 'Schema',
+  TABLE = 'Table',
+  ENUM = 'Enum',
+  ENUM_MEMBER = 'Enum field',
+  COLUMN = 'Column',
+  TABLE_GROUP = 'TableGroup',
+}
+
+export interface NodeSymbol {
+  kind: SymbolKind;
+  symbolTable?: SymbolTable;
+  declaration?: SyntaxNode;
+}
+
+// A symbol for a schema, contains the schema's symbol table
+export class SchemaSymbol implements NodeSymbol {
+  kind: SymbolKind.SCHEMA = SymbolKind.SCHEMA;
+
+  symbolTable: SymbolTable;
```

Comment:
The meaning of `symbolTable` here is quite vague. Is it used for storing all declarations in the schema's scope or storing the schema's members?

Reply:
so I think it's better to rename it to `members`